src/Controller/HealthController.php

```diff
@@ -51,13 +51,48 @@ public function performanceStatus(): JsonResponse
         return new JsonResponse($collection);
     }
 
-    #[Route(path: '/health-ping/status', name: 'api.frosh.tools.health-ping.status', methods: ['GET'])]
-    public function pingStatus(): JsonResponse
+    #[Route(path: '/health/check', name: 'api.frosh.tools.health-ping.status', methods: ['GET'])]
+    public function healthCheck(): JsonResponse
     {
-        return $this->cacheObject->get('health-ping', function (ItemInterface $cacheItem) {
+        $healthStatus = $this->getHealthCachedStatus();
+
+        return new JsonResponse(['state' => $healthStatus->getState()]);
+    }
+
+    private function getHealthStatus(): HealthCollection
+    {
+        $collection = new HealthCollection();
+        foreach ($this->healthCheckers as $checker) {
+            $checker->collect($collection);
+        }
+
+        return $collection;
+    }
+
+    private function getHealthCachedStatus(): HealthCollection
+    {
+        return $this->cacheObject->get('health-check', function (ItemInterface $cacheItem) {
             $cacheItem->expiresAfter(59);
 
-            return $this->status();
+            $healthStatus = $this->getHealthStatus();
+
+            if ($this->configService->getBool('FroshTools.config.healthLogger')) {
+                foreach ([SettingsResult::ERROR, SettingsResult::WARNING] as $state) {
+                    $healthStatus = $healthStatus->filter(function (SettingsResult $result) use ($state) {
+                        return $result->state === $state;
+                    });
+
+                    if ($healthStatus->count() > 0) {
+                        if ($state === SettingsResult::ERROR) {
+                            $this->froshToolsLogger->error('Health check', ['results' => $healthStatus]);
```

Comment:
this will log it every x minutes sounds useless

Reply:
I thought: when would be a better time? 🙈 always is really dump.